.github/CODEOWNERS

```diff
@@ -11,7 +11,7 @@
 /.github/CODEOWNERS @edolstra
 
 # GitHub actions
-/.github/workflows @Mic92 @zowoq
+/.github/workflows @NixOS/Security @Mic92 @zowoq
```

Comment:
@NixOS/Security doesn't seem to exist?

Listing a team here is a no op unless that team has commit access.

https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

> The people you choose as code owners must have write permissions for the repository. When the code owner is a team, that team must have write permissions, even if all the individual members of the team already have write permissions directly, through organization membership, or through another team membership.

Reply:
Thanks, the team was private for some reason. It is public now.